Core/Core/Dashboard/K5/View/Grades/K5GradesView.swift

```diff
@@ -68,13 +69,17 @@ struct K5GradesView: View {
                             .font(.bold24)
                             .foregroundColor(.licorice)
                     })
+                    .accessibility(label: Text("Select grading period", bundle: .core))
+                    .accessibility(hint: Text(gradeSelectorOpen ? "Open": "Closed", bundle: .core) +
+                                   Text(", \(viewModel.currentGradingPeriod.title ?? "")"))
```

Comment:
Announcing the currently selected grading period as a hint is a good idea! Maybe we could add some explanation on why we are announcing it. "Active grading period, grading period 1" or "Grading period 1, selected".

Reply:
Edit changes